services/core-web/src/components/noticeOfWork/applications/referals/NOWApplicationReviewsTable.js

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import { Table, Button, Icon, Popconfirm } from "antd";
+import { PropTypes } from "prop-types";
+import { connect } from "react-redux";
+import CustomPropTypes from "@/customPropTypes";
+import { getNoticeOfWorkApplicationApplicationReviewTypeHash } from "@/selectors/staticContentSelectors";
+import AuthorizationWrapper from "@/components/common/wrappers/AuthorizationWrapper";
+import * as Permission from "@/constants/permissions";
+import { EDIT_OUTLINE_VIOLET } from "@/constants/assets";
+import LinkButton from "@/components/common/LinkButton";
+import { downloadFileFromDocumentManager } from "@/utils/actionlessNetworkCalls";
+/**
+ * @constant ReviewNOWApplication renders edit/view for the NoW Application review step
+ */
+
+const propTypes = {
+  // eslint-disable-next-line
+  noticeOfWorkReviews: PropTypes.arrayOf(CustomPropTypes.NOWApplicationReview).isRequired,
+  noticeOfWorkReviewTypesHash: PropTypes.objectOf(PropTypes.string).isRequired,
+
+  handleDocumentDelete: PropTypes.func.isRequired,
+  handleDelete: PropTypes.func.isRequired,
+  openEditModal: PropTypes.func.isRequired,
+  handleEdit: PropTypes.func.isRequired,
+};
+
+const columns = [
+  {
+    title: "Type",
+    dataIndex: "now_application_review_type",
+    key: "now_application_review_type",
+  },
```

Comment:
These should all have a render so they will have the correct styling when responsive.

```
render: (text) => (
      <div title="Type">
      {text}
      </div>
    ),
```

Reply:
added.